Description: render node-request optional
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-01-11

--- a/lib/install.js
+++ b/lib/install.js
@@ -7,7 +7,6 @@
 const crypto = require('crypto')
 const log = require('npmlog')
 const semver = require('semver')
-const request = require('request')
 const processRelease = require('./process-release')
 const win = process.platform === 'win32'
 const getProxyFromURI = require('./proxy')
@@ -394,6 +393,7 @@
 }
 
 function download (gyp, env, url) {
+  const request = require('request')
   log.http('GET', url)
 
   var requestOpts = {
